insurance/README.md

```diff
@@ -70,10 +70,12 @@
 1. Name : insurance-training-data 
    - This is DKube local dataset, created by the training/retraining pipeline above
 2. Type : csv
-3. Version: v2
-4. Add transformer script: https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance/transform-data.py
-5. Note: Download the script in your setup and then add it by browsing.
-6. Save training data.
+3. If training data source is SQL
+      - Add query: `select * from insurance` (the table name can be different)
+4. If training Version: v2 
```

Comment:
This line doesn't fit with the flow. Are you saying, if v2 Add Transformer?

Reply:
It's moved up.